examples/vsphere/sample-terraform-dhcp.tfvars

```diff
@@ -122,19 +121,22 @@ node_pools = {
   },
   compute = {
     count       = 1
-    cpus        = 16
-    memory      = 131072
+    cpus        = 4
+    memory      = 16384
```

Comment:
I thought compute needed 128G of memory to run items there. Interesting.

Reply:
That's based off their testing and it also lines up with the SAS Doc that states 16 is the min for compute for a "SAS Viya" deployment
https://go.documentation.sas.com/doc/en/itopscdc/v_052/itopssr/n0ampbltwqgkjkn1j3qogztsbbu0.htm#n11a8r150qgkcon17txx5r526sd9